### PR TITLE
Implement multi-threading of comments

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
screen shot 2017-03-09 at 10 32 23 am

**Comment by nathanbegbie** [comment]
screen shot 2017-03-09 at 10 32 23 am

**Comment by nathanbegbie** [comment]
screen shot 2017-03-09 at 10 32 34 am

**Comment by nathanbegbie** [comment]
screen shot 2017-03-09 at 10 32 53 am

File: molo/commenting/templatetags/molo_commenting_tags.py [hunk 1]
```diff
@@ -34,17 +36,26 @@ def get_molo_comments(parser, token):
...
+                                   keywords[6], keywords[8])
     return GetMoloCommentsNode(keywords[2], keywords[4])
 
 
 class GetMoloCommentsNode(template.Node):
 
-    def __init__(self, obj, variable_name, limit=5):
+    def __init__(self, obj, variable_name, limit=5, child_limit=0):
         self.obj = obj
+        print(obj)
```
Comment (miltontony) [inline molo/commenting/templatetags/molo_commenting_tags.py]: stray print

**Comment by miltontony** [comment]
👍 🍰 🍬 once travis and conflicts ;)

**Pull request merged**